backlogger.py

```diff
@@ -89,33 +93,30 @@ def issue_reminder(conf, poo, poo_reminder_state):
             if priority == "Low" and poo_reminder_state['has_repeat_reminder']:
                 print("Skipping priority update for {}, already at lowest".format(poo["id"]))
                 return
-            _update_issue_priority(poo["id"], priority, poo_reminder_state)
+            _update_issue_priority(poo["id"], priority, poo_reminder_state, msg)
             return
-        _send_first_reminder(poo["id"], priority, conf)
+        _send_first_reminder(poo["id"], msg)
 
 
-def _send_first_reminder(poo_id, priority_current, conf):
-    msg = reminder_text.format(priority=priority_current, url=data["url"])
-    if "comment" in conf:
-        msg = conf["comment"]
+def _send_first_reminder(poo_id, msg):
     print("Writing reminder for {}".format(poo_id))
     url = "{}/{}.json".format(data["web"], poo_id)
     json_rest("PUT", url, {"issue": {"notes": msg}})
 
 
-def _update_issue_priority(poo_id, priority_current, poo_reminder_state):
+def _update_issue_priority(poo_id, priority_current, poo_reminder_state, msg):
     if poo_reminder_state['has_repeat_reminder'] and (poo_reminder_state['last_reminder'] + slo_priorities[priority_current]["period"]) < present:
-            
-        msg = "No response to reminder. Reducing priority from {} to next lower {} for {}"
-        print(msg.format(priority_current,
+        note = "No response to reminder. Reducing priority from {} to next lower {} for {}"
+        print(note.format(priority_current,
                          slo_priorities[priority_current]["next_priority"]["name"],
                          poo_id))
         url = "{}/{}.json".format(data["web"], poo_id)
+        msg = " ".join(update_slo_text.format(
```

Comment:
If I execute this I think it's not what we want :D
```
>>> update_slo_text = "The ticket will be set to the next lower priority **{priority}**."
>>> msg = " ".join(update_slo_text.format(priority=23))
>>> msg
'T h e   t i c k e t   w i l l   b e   s e t   t o   t h e   n e x t   l o w e r   p r i o r i t y   * * 2 3 * * .'
```

Reply:
And @b10n1k please see if you can add test coverage - since I mistakenly thought this was covered, which is why I thought it must still work